doc/main.html

```diff
@@ -229,6 +231,10 @@ <h4>Overall structure</h4>
               <td><a>Annex section</a></td>
               <td>0 or more</td>
             </tr>
+            <tr>
+              <td><a>non-prose elements section</a></td>
```

Comment:
To be in line with AG-16:
```suggestion
              <td><a>Additional elements section</a></td>
```